diff --git a/node_modules/react-native-simple-toast/react-native-simple-toast.podspec b/node_modules/react-native-simple-toast/react-native-simple-toast.podspec
index 7775bf9..75ecba5 100644
--- a/node_modules/react-native-simple-toast/react-native-simple-toast.podspec
+++ b/node_modules/react-native-simple-toast/react-native-simple-toast.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
 
   s.authors      = package['author']
   s.homepage     = package['homepage']
-  s.platform     = :ios, "9.0"
+  s.platform     = :ios, "12.4"
 
   s.source       = { :git => "https://github.com/vonovak/react-native-simple-toast.git", :tag => "v#{s.version}" }
   s.source_files  = "ios/**/*.{h,m}"
